AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+buykers.com##.m-popup.m-popup__cookies
```

Comment:
`buykers.com##.m-popup__cookies` is enough, please change

Reply:
@Sergey-Lyapin Fixed.